Original patch from: ../4.3.2/320-alpha-signal_h.patch

-= BEGIN original header =-
-= END original header =-

diff -durN gcc-4.3.3.orig/gcc/config/alpha/linux-unwind.h gcc-4.3.3/gcc/config/alpha/linux-unwind.h
--- gcc-4.3.3.orig/gcc/config/alpha/linux-unwind.h	2006-10-26 21:31:09.000000000 +0200
+++ gcc-4.3.3/gcc/config/alpha/linux-unwind.h	2009-01-27 22:25:49.000000000 +0100
@@ -29,6 +29,9 @@
 /* Do code reading to identify a signal frame, and set the frame
    state data appropriately.  See unwind-dw2.c for the structs.  */
 
+/* Don't use this if inhibit_libc is set
+   The build for this target will fail trying to include missing headers */
+#ifndef inhibit_libc
 #include <signal.h>
 #include <sys/ucontext.h>
 
@@ -80,3 +83,4 @@
   fs->retaddr_column = 64;
   return _URC_NO_REASON;
 }
+#endif /* inhibit_libc */
